lms/envs/common.py

```diff
@@ -5384,3 +5384,6 @@ def _make_locale_paths(settings):  # pylint: disable=missing-function-docstring
 #### django-simple-history##
 # disable indexing on date field its coming from django-simple-history.
 SIMPLE_HISTORY_DATE_INDEX = False
+
+#### Event bus publishing ####
+EVENT_BUS_PRODUCER_CONFIG = {}
```

Comment:
1. Have we set this locally to ensure it doesn't cause an issue?
2. Maybe we should have another comment and/or link to a ticket for future rollout so no one tries to set this?